order/supplier/receive/simple-order-event.md

```diff
@@ -0,0 +1,249 @@
+---
+description: How to receive a simple order sent by the buyer.
+---
+
+# Receive an order
+
+Tradecloud will send a purchase order, either new or updated, to the supplier when an order event has been triggered.
+
+## `simpleOrderEvent`
+
+This page assumes you are using the webhook with the simple delivery schedule in the `simpleOrderEvent`.
+
+* `orderId`: the Tradecloud order identifier.
```

Comment:
Seeing this long list of all properties, written out to the deepest level; are we sure we don't want to improve the API docs instead? 
I think it will increase maintainability a lot if we only mention a high-level summary and only the functional considerations in a page like this..

Reply:
I have added this to https://tradecloud.atlassian.net/browse/TC-9548